af_pipeline/af_input/msa.py

```
@@ -0,0 +1,353 @@
+        ## Returns:
+
+        - **dict**:<br />
+            Dictionary containing the response from the MMSeqs2 API.
+        """
+
+        query = ""
+        for header, seq in sequences.items():
+            query += f">{header}\n{seq}\n"
+
+        _url = MMSeqs2API.get_ticket_url(use_pairing=self.use_pairing)
+        res = requests.post(_url, data={"q": query, "mode": mode})
```

Ruff (0.16.3)

[error] Probable use of `requests` call without timeout

(S113)